### PR TITLE
Fix broken footer license link.

### PR DESCRIPTION
**Pull request by montehurd**
@josve05a

### CODE REVIEW

File: Wikipedia/Code/WebViewController.m [hunk 1]
```diff
@@ -290,8 +290,7 @@ - (WMFArticleFooterView*)footerLicenseView {
         @weakify(self);
         [_footerLicenseView.showLicenseButton bk_addEventHandler:^(id sender) {
             @strongify(self);
-            MWKSite* site = [[MWKSite alloc] initWithDomain:WMFDefaultSiteDomain language:@"en"];
-            [self.delegate webViewController:self didTapOnLinkForTitle:[site titleWithString:WMFLicenseTitleOnENWiki]];
```
Comment (josve05a) [inline Wikipedia/Code/WebViewController.m]: Just a though,
'@"by-sa/3.0";'
and
`MWKSite* site = [[MWKSite alloc] initWithDomain:WMFDefaultSiteDomain language:@"creativecommons:"];
[self.delegate webViewController:self didTapOnLinkForTitle:[site titleWithString:WMFLicenseTitleOnENWiki]];`
would not work then?

Comment (montehurd) [inline Wikipedia/Code/WebViewController.m]: @josve05a I tried it. No it doesn't work. Out of curiosity, what was the problem with the former page this was linking to?

Comment (josve05a) [inline Wikipedia/Code/WebViewController.m]: It's an internal page on enwp, which is not translated in to multiple languages (officially). It's true that en.wp (desktop) links to it, but mobile does not, and I think the app should use same standard and wordings as mobile.

Comment (montehurd) [inline Wikipedia/Code/WebViewController.m]: ah gotcha.

**Pull request merged**